modules/org_policy_v2/README.md

```diff
@@ -0,0 +1,122 @@
+# Google Cloud Organization Policy Terraform Module
+
+This Terraform module makes it easier to manage [organization policies](https://cloud.google.com/resource-manager/docs/organization-policy/overview) for your Google Cloud environment, particularly when you want to have exclusion rules. This module will allow you to set a top-level org policy and then disable it on individual projects or folders easily. This module allows **_conditional policy enforcements based on the latest version_** of the organization policies API.
+
+Organization Policies are of two types `boolean` and `list`. See the documentation below for more information on:
+ - [Customizing policies for boolean constraints](https://cloud.google.com/resource-manager/docs/organization-policy/creating-managing-policies#boolean_constraints)
+
+
+## Compatibility
+This module is meant for use with Terraform 0.13+ and tested using Terraform 1.0+. If you find incompatibilities using Terraform >=0.13, please open an issue.
+ If you haven't
+[upgraded](https://www.terraform.io/upgrade-guides/0-13.html) and need a Terraform
+0.12.x-compatible version of this module, the last released version
+intended for Terraform 0.12.x is [v4.0.0](https://registry.terraform.io/modules/terraform-google-modules/-org-policy/google/v4.0.0).
+
+## Usage
+Example usage is included in the [examples](./examples/org_policy_v2) folder, but simple usage is as follows:
+
+```hcl
+module "gcp_org_policy" {
+  source            = "terraform-google-modules/org-policy/google//modules/org_policy_v2"
+  version           = "~> 4.0.0"
+
+  org_policies = {
+    "gcp-org-policy-list-01" = {
+      constraint       = "constraint name" # constraint identifier without constriants/ prefix
+      type             = "list"
+      organization_id  = "123456789"       # either of org id, folder id or project id
```

Comment:
This seems repetitive, can't we have a global default and optional overrides?

Reply:
Each `policy` needs the root to be defined, hence thought of adding it part of each `map`. Let me rethink on this.